.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,24 @@
+name: Question
```

Comment:
I'd add a platform option here as well (could be optional). In many cases questions are for specific platforms

Reply:
added